### PR TITLE
Device control violations fix 

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8000

## Description
Cortex XDR device control violations" playbook- Added a task to check if the email was retrived from the AD before attempting to send an email. 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/6_1_13.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR device control violations
+
+- Added condition task to check if the email was retrieved from the Active Directory.
```
Comment (ssokolovich) [inline Packs/CortexXDR/ReleaseNotes/6_1_13.md:6]:
```suggestion
- Added a conditional task to check if the user's email was retrieved from the Active Directory. 
```

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations.yml [hunk 2]
```diff
@@ -496,6 +496,51 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: 1a497783-b55d-4aec-8764-2722b3d3216d
+    type: condition
+    task:
+      id: 1a497783-b55d-4aec-8764-2722b3d3216d
+      version: -1
+      name: Is the email retrieved?
```
Comment (ssokolovich) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations.yml:508]:
```suggestion
      name: Is the user's email retrieved?
```

**Review by ssokolovich — APPROVED** [review]

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations_README.md [hunk 3]
```diff
@@ -3,38 +3,47 @@ All the collected data will be displayed in the XDR device control incident layo
...
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
-| TimeStamp | Timestamp in relative date format for query device control events from Cortex XDR. |  | Optional |
-| EndpointID | XDR endpoint ID to filter results for. |  | Optional |
-| Hostname | Hostname to filter results for. |  | Optional |
-| IPAddress | IP address to filter results for. |  | Optional |
+| TimeStamp | Timestamp in relative date format for query device control events<br/>from Cortex XDR.<br/>For example "1 day", "3 weeks". |  | Optional |
+| EndpointID | Comma-separated list of endpoint IDs to filter results for. |  | Optional |
+| Hostname | Comma-separated list of hostnames to filter results for. |  | Optional |
+| IPAddress | Comma-separated list of endpoints IP Addresses to filter results for. |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_device_control_violations_README.md:36]:
```suggestion
| IPAddress | Comma-separated list of endpoints IP addresses to filter results for. |  | Optional |
```

File: Packs/CortexXDR/ReleaseNotes/6_1_13.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Cortex XDR device control violations
+
+- Added a conditional task to check if the user's email was retrieved from the Active Directory.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXDR/ReleaseNotes/6_1_13.md:6]:
```suggestion
Added a conditional task to check if the user's email was retrieved from Active Directory. 
```

**Comment by ShirleyDenkberg** [comment]
@ssokolovich Doc review completed.

**Pull request merged**